### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by rannick**
Release 2.2.0

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]
Reviewed most of the PRs and also local tests on larger data are running fine now 👍🏻

**Comment by maxulysse** [comment]
Amazing work

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,43 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.2.0 - [2022/03/06]
```
Comment (apeltzer) [inline CHANGELOG.md:6]:
```suggestion
## v2.2.0 - [2022/03/13]
```

**Review by maxulysse — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?logo=Amazon%20AWS)](https://nf-co.re/rnafusion/results)
-[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8)](https://doi.org/10.5281/zenodo.XXXXXXX)
-
-[![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A521.10.3-23aa62.svg)](https://www.nextflow.io/)
-[![run with conda](http://img.shields.io/badge/run%20with-conda-3EB049?logo=anaconda)](https://docs.conda.io/en/latest/)
-[![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?logo=docker)](https://www.docker.com/)
-[![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg)](https://sylabs.io/docs/)
+[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/rnafusion/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
```
[the remark this answers is not in the document]
Comment (apeltzer) [reply README.md:3]: [DOI](https://zenodo.org/badge/latestdoi/151721952)

Would be the right one

**Comment by rannick** [comment]
I will need this one merged too for aws full tests: https://github.com/nf-core/rnafusion/pull/323

File: conf/test_full.config [hunk 3]
```diff
@@ -3,21 +3,16 @@
...
 
-    // Input data
-    genome                 = 'GRCh38'
-    input                  = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnafusion/testdata/human/samplesheet_valid.csv'
-    genomes_base           = "${params.outdir}/references"
-    all                    = true
+    // Input data for full size test
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnafusion/testdata/human/samplesheet_valid.csv'
 
-}
```
[the remark this answers is not in the document]
Comment (rannick) [reply conf/test_full.config:23]: Yes! [cc85523](https://github.com/nf-core/rnafusion/pull/320/commits/cc85523bb37da771e72e39f2437db82de1a721e4)

File: nextflow.config [hunk 4]
```diff
@@ -14,11 +14,19 @@ params {
...
 
     // MultiQC options
     multiqc_config             = null
     multiqc_title              = null
+    multiqc_logo               = null
     max_multiqc_email_size     = '25.MB'
+    multiqc_methods_description = null
 
     // Genome
     genome                     = 'GRCh38'
```
[the remark this answers is not in the document]
Comment (rannick) [reply nextflow.config:27]: Removed previous lines

File: nextflow.config [hunk 5]
```diff
@@ -14,11 +14,19 @@ params {
     build_references           = false
     cosmic_username            = null
     cosmic_passwd              = null
+    qiagen                     = false
+
+    // References
+    genome                     = null
+    igenomes_base              = 's3://ngi-igenomes/igenomes'
+    igenomes_ignore            = true
```
[the remark this answers is not in the document]
Comment (rannick) [reply nextflow.config]: Removed

File: nextflow.config [hunk 6]
```diff
@@ -101,13 +119,21 @@ params {
...
     max_time                   = '240.h'
 }
 
+// Load igenomes.config if required
+if (!params.igenomes_ignore) {
+    includeConfig 'conf/igenomes.config'
+} else {
+    params.genomes = [:]
+}
+
```
[the remark this answers is not in the document]
Comment (rannick) [reply nextflow.config]: Removed

**Review by apeltzer — APPROVED** [review]

**Comment by apeltzer** [comment]
Merged that one for you, looks good now for me and everything that @maxulysse found was addressed now too 👍🏻

**Review by maxulysse — APPROVED** [review]

**Pull request merged**